src/lib/Libnet/net_server.c

```diff
@@ -549,26 +605,7 @@ wait_request(time_t waittime)
 				}
 			}
 #endif
-			idx = connection_find_actual_index(em_fd);
-			if (idx < 0) {
-				return -1;
-			}
-
-			svr_conn[idx]->cn_lasttime = time(NULL);
-
-			if (svr_conn[idx]->cn_active != Primary && svr_conn[idx]->cn_active != RppComm && svr_conn[idx]->cn_active
-					!= Secondary) {
-
-				if (!(svr_conn[idx]->cn_authen & PBS_NET_CONN_AUTHENTICATED)) {
-
-					if (engage_authentication(svr_conn[idx]) == -1) {
-						close_conn(em_fd);
-						continue;
-					}
-				}
-			}
-
-			svr_conn[idx]->cn_func(svr_conn[idx]->cn_sock);
+			process_socket(em_fd);
```

Comment:
Validate if process_socket() returns failure.

Reply:
Yes. You can log a message at debug level.

Reply:
done